app/models/label_template.rb

```diff
@@ -12,14 +12,11 @@ def self.for_type(class_type, barcodetype = 'ean13')
 
     templates = where(template_type: type)
 
-    unless templates.count.positive?
```

Comment:
`exists?` is usually better than `count.positive?` as it runs a query more akin to the following:
```sql
SELECT  1 AS one FROM `label_templates` WHERE `label_templates`.`template_type` = 'Plate' LIMIT 1
```
Rather than

```sql
SELECT COUNT(*) FROM `label_templates` WHERE `label_templates`.`template_type` = 'Plate'
```

Which is significantly faster with larger tables.

Here however I'm using `blank?` which will actually pull back the templates themselves. In this case this actually saves us a trip to the database, as we're going to immediately use the array in the next step. (As this exception case should be exceedingly rare)